RocketChat/Rocket.Chat
# fix: File attachments have no content when exporting room messages as file

## Proposed changes (including videos or screenshots)


 - Wait for file content to be fetched and stored before zipping and sending message export (on Amazon S3, Webdav and Google Storage).



## Issue(s)


## Steps to test or reproduce

1. Setup your workspace to use Amazon S3, Google Storage or Webdav as storage type;
2. Send some file attachments (eg images) in a room;
3. Go to the room options (three dots) > Export messages and choose the "Export as file" method. Select any time period that includes the file attachments sent.
4. Check your e-mail and download the linked file.

**_Expected behavior:_** the attachments should be available inside the assets directory, and all of their content should be visible just as in RC.
**_Current behavior:_** all files are created and available in the assets directory, but they have no content (file sizes are all zero).

## Further comments

SUP-358
